proposals/exp-show-target.md

```diff
@@ -0,0 +1,214 @@
+
+It's also likely the busiest DVC output:
+
+```console
+┏━━━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━┳━━━━━━━━━┳━━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━━━━━┳━━━━━━━━━━━━┳━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━━━━━━━━┳━━━━
+┃ Experiment     ┃ Created      ┃     auc ┃ prepare.split ┃ prepare.seed ┃ featurize.max_features ┃ featurize.ngrams ┃ train.seed ┃ train.n_estimators ┃ topic_modeling.seed ┃ top
+┡━━━━━━━━━━━━━━━━╇━━━━━━━━━━━━━━╇━━━━━━━━━╇━━━━━━━━━━━━━━━╇━━━━━━━━━━━━━━╇━━━━━━━━━━━━━━━━━━━━━━━━╇━━━━━━━━━━━━━━━━━━╇━━━━━━━━━━━━╇━━━━━━━━━━━━━━━━━━━━╇━━━━━━━━━━━━━━━━━━━━━╇━━━━
+│ workspace      │ -            │ 0.51625 │ 0.2           │ 20170428     │ 1500                   │ 2                │ 20170428   │ 10                 │ 20170428            │ 10
+│ topic_modeling │ 04:41 PM     │ 0.51625 │ 0.2           │ 20170428     │ 1500                   │ 2                │ 20170428   │ 10                 │ 20170428            │ 10
+│ master         │ Feb 17, 2021 │ 0.51625 │ 0.2           │ 20170428     │ 1500                   │ 2                │ 20170428   │ 10                 │ -                   │ -
+│ └── exp-56d11  │ 04:40 PM     │ 0.51625 │ 0.2           │ 20170428     │ 1500                   │ 2                │ 20170428   │ 10                 │ 20170428            │ 10
+│ 263732a        │ Feb 11, 2021 │  0.5674 │ 0.2           │ 20170428     │ 1500                   │ 2                │ -          │ 100                │ -                   │ -
+│ b297969        │ Feb 11, 2021 │  0.5674 │ 0.2           │ 20170428     │ 1500                   │ 2                │ -          │ 100                │ -                   │ -
+│ 4edd930        │ Feb 11, 2021 │  0.5674 │ 0.2           │ 20170428     │ 1500                   │ 2                │ -          │ 100                │ -                   │ -
+│ 72ed9cd        │ Nov 15, 2020 │  0.5674 │ 0.2           │ 20170428     │ 1500                   │ 2                │ 20170428   │ 50                 │ -                   │ -
+│ ├── exp-47dfa  │ Feb 17, 2021 │ 0.51625 │ 0.2           │ 20170428     │ 1500                   │ 2                │ 20170428   │ 10                 │ -                   │ -
+│ └── exp-44136  │ Feb 17, 2021 │  0.5674 │ 0.2           │ 20170428     │ 1500                   │ 2                │ 20170428   │ 50                 │ -                   │ -
+│ f8e9d93        │ Nov 15, 2020 │ 0.54175 │ 0.2           │ 20170428     │ 1500                   │ 2                │ 20170428   │ 50                 │ -                   │ -
+```
+
+This table view is from a simplified example and is still missing several
+columns, so this view can become cluttered quickly. Eliminating irrelevant
+information is a priority.
+
+There are several ways in which the included parameters might be irrelevant:
+* Parameters files may include values that aren't tracked by DVC at all (for
+  example, debug level).
```

Comment:
Same for metrics.

Reply:
I think that in case of metrics we don't have this problem: https://github.com/iterative/dvc/issues/5451#issuecomment-796674685
because metrics are files while params are loaded as files while they should be loaded as file entries/lines.

Reply:
So is the bullet about metrics correct?

> The command's output currently includes: ...
* All values in any referenced metrics files in any DVC stage.

Reply:
All metrics are outputs from a DVC stage, which means that:
* Unlike parameters, metrics are always tracked by some DVC stage.
* Metrics still may be irrelevant to a particular DVC stage, as mentioned in the bullets below.